cmake/modules/FindX10.cmake

```diff
@@ -0,0 +1,11 @@
+file(TO_CMAKE_PATH "${X10_TOOLCHAIN_PATH}" X10_CMAKE_TOOLCHAIN_PATH)
```

Comment:
Please don't do this.  You can use `get_filename_component` on `${CMAKE_C_COMPILER}` and use that to compute the path you want and use that as a hint, but please don't base this behaviour on this random variable.  There is a standard behaviour expected of the file package (https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html).